engine/runtime-parser/src/main/java/org/enso/compiler/core/TreeToIr.java

```diff
@@ -1921,6 +1921,9 @@ private List<Name> qualifiedNameSegments(Tree t, boolean generateId) throws Synt
       names = join(sanitizeName(buildName(app.getRhs(), generateId)), names);
       t = app.getLhs();
     }
+    if (t instanceof Tree.Call call) {
+        t = call.getValue();
```

Comment:
- the intentions of [call nodes](https://github.com/enso-org/design/pull/60) seem to be good
- now when I "get them"
- let's try the approach